tests/Optimizer/SpecTest.php

```diff
@@ -52,6 +54,8 @@ final class SpecTest extends TestCase
         'PreloadHeroImage - disable_via_param'             => 'see https://github.com/ampproject/amp-toolbox-php/issues/55',
 
         'MinifyHtml - minifies_inline_amp-script'          => 'see https://github.com/ampproject/amp-toolbox-php/issues/260',
+
+        'AmpStoryCssOptimizer - add_aspect_ratio_to_style' => '',
```

Comment:
@schlessera The reason I'm ignoring the test here because the new inline css append to existing css by `\AmpProject\Dom\Element::addInlineStyle()` method, but according to the spec expected output, we need to prepend the new style. So, I need your suggestion about the possible solution for this. Should I keep it ignoring or should I modify the `Element::addInlineStyle()` method or anything else?

Reply:
@ediamin I'd say we add an optional argument `$prepend = false` to the `\AmpProject\Dom\Element::addInlineStyle()` method. This way, we can choose to prepend, but we don;t break existing code.

Reply:
Thanks. I updated the code.